src/app/components/static-footer/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import autobind from 'autobind-decorator';
+
+import styles from './styles.css';
+import image from './giphy.gif';
+
+// const cx = classNames.bind(styles);
+
+class StaticFooter extends Component {
+  buildTitle(isTrending, totalResults, count, offset) {
+    if (isTrending) {
+      return `Top Trending GIFs!`;
+    }
+    const totalPages = Math.ceil(totalResults / count);
+    const currentPage = offset / count + 1;
+    return `Showing page ${currentPage} of ${totalPages}`;
+  }
+
+  @autobind
+  decrement() {
+    const {currentOffset, count, changeOffset, navPrevEnabled} = this.props;
+    if (!navPrevEnabled) {
+      return;
+    }
+    const offset = currentOffset >= count ? currentOffset - count : 0;
+    changeOffset(offset);
+  }
+
+  @autobind
+  increment() {
+    const {currentOffset, count, changeOffset, navForwardEnabled} = this.props;
+    if (!navForwardEnabled) {
+      return;
+    }
+    const offset = currentOffset + count;
+    changeOffset(offset);
+  }
+
+  render() {
+    const {count, currentOffset, isTrending, navForwardEnabled, navPrevEnabled, showNav, totalResults} = this.props;
+    const title = this.buildTitle(isTrending, totalResults, count, currentOffset);
+    const prevClass = classNames({
+      [`${styles.button}`]: true,
+      [`${styles.buttonDisabled}`]: !navPrevEnabled
+    });
+    const nextClass = classNames({
+      [`${styles.button}`]: true,
+      [`${styles.buttonDisabled}`]: !navForwardEnabled
+    });
+    return (
+      <div className={styles.footer}>
+        {showNav && (
+          <div className={styles.side}>
+            <div
+              className={prevClass}
+              onClick={this.decrement}
+            >
```

Comment:
Purely stylistic nonsense: to me breaking divs up on 4 lines like this is a bit over the top for 2 attributes.

![bloop de blah blah](https://media1.giphy.com/media/L0aWDywDu1ziw/giphy.gif?fingerprint=e1bb72ff59a2fbbf524b4e3359f456fc)

Reply:
Good call. This was before I had the 'prevClass' var and made it way too long for one line.